packer/scripts/ubuntu/infiniband.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+echo " ********************************************************************************** "
+echo " *                                                                                * "
+echo " *     INFINIBAND CONFIGURAION                                                    * "
+echo " *                                                                                * "
+echo " ********************************************************************************** "
+set -ex
```

Comment:
is this script valid for ubuntu18 & 20 & 22? if not let's make forks

Reply:
also propose to add this to the PG repo for ubuntu-hpc

Reply:
agree with both points. at present we only have 18.04 image and will need to verify all scripts for 20 and 22.